### DISCUSSION TITLE
Will laragon receive updates to PHP 8.2 and Laravel 11?

### DISCUSSION THREAD

**Discussion by Lexian-droid** [leokhoa/laragon#719] [Q&A] [answered]
Hello, I've realized Laravel seems to have released the next iteration of their framework, and so I was curious if Laragon would be upgrading as well, I use Laragon for Laravel projects, and if I ever do upgrade my project, I'd like to know if Laragon will be able to continue to support my development, as I've been using it for a while now in place of MAMP.

**Comment by marcoguzman0** [comment, marked as answer]
You're actually capable to create and use Laravel 11 apps in Laragon, just be sure to update this two packages manually:

- PHP to 8.3.3 (make sure to enable the "pdo_sqlite" extension in php.ini)
- Apache to httpd-2.4.58

![image](https://github.com/leokhoa/laragon/assets/51141509/11151e03-252d-491f-b821-bb3dbcf15f46)

**Reply by Lexian-droid** [reply]
Ahh I see, thanks for the information! I will definitely do this! 👍🏻

**Comment by Pen-y-Fan** [comment]
Laragon 6.0 supports PHP 8.2, the minimum requirement for Laravel 11.

If you want to upgrade to PHP 8.3 see my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/), you will also need to upgrade [Apache to the latest version](https://www.apachelounge.com/download/).

Laragon's quick app for Laravel will always install the latest version. (Menu > Quick app > Laravel)

**Reply by Lexian-droid** [reply]
Oh okay thanks!

**Reply by ThorondorManwe** [reply]
Thanks :)

**Comment by aymonyz** [comment]
شكرا  لقد افادني ردك جدأ

**Comment by maullana19** [comment]
for add another version, download php binary add put into bin folder php.
